<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>611</width>
    <height>392</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>331</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>611</width>
      <height>331</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>客房查询</string>
     </attribute>
     <widget class="QTableView" name="tvPriceInfo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>591</width>
        <height>121</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tvRoomInfo">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>256</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>21</width>
        <height>121</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;今&lt;/p&gt;&lt;p&gt;日&lt;/p&gt;&lt;p&gt;房&lt;/p&gt;&lt;p&gt;价&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>120</y>
        <width>21</width>
        <height>151</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;房&lt;/p&gt;&lt;p&gt;间&lt;/p&gt;&lt;p&gt;信&lt;/p&gt;&lt;p&gt;息&lt;/p&gt;&lt;p&gt;表&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>151</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>房间类型</string>
      </property>
      <widget class="QRadioButton" name="rb6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>总统套房</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb1">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>商务套房</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>豪华单人间</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>豪华双人间</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>标准单人间</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="rb4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>标准双人间</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>登记入住</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>391</width>
        <height>261</height>
       </rect>
      </property>
      <property name="title">
       <string>信息填写</string>
      </property>
      <widget class="QLineEdit" name="leID">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leUserName">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>221</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leRoomID">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leNumOfNig">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>130</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>100</y>
         <width>3</width>
         <height>61</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
      <widget class="Line" name="line_2">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>0</y>
         <width>301</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="Line" name="line_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>371</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>50</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>证件号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>80</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>姓　名：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>110</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>房间号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>入住天数：</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbSubmit">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbCancel">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>210</y>
         <width>99</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>退房结算</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>491</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>欢迎下次光临</string>
      </property>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>证件号：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>70</y>
         <width>67</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>入住信息</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>230</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>应缴费用：</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_10">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>200</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>实际入住天数：</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="leOut_ID">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>221</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="leRealdate">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>200</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lePay">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>113</width>
         <height>27</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pbOut_Search">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>30</y>
         <width>99</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pbCheckout">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>230</y>
         <width>99</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>退房</string>
       </property>
      </widget>
      <widget class="Line" name="line_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>60</y>
         <width>411</width>
         <height>16</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QTableView" name="tvQuery">
       <property name="geometry">
        <rect>
         <x>35</x>
         <y>90</y>
         <width>401</width>
         <height>101</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>记录查询</string>
     </attribute>
     <widget class="QTableView" name="tvCheckin">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>511</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tvCheckout">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>180</y>
        <width>511</width>
        <height>111</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
